.github/CONTRIBUTING.md

```diff
@@ -65,11 +68,186 @@ Pull Requests complies with these rules:
+
+- git commandline tool
+  - for running "git commands" with
+- Git SCM (source code management) format
+  - Git repositories are directories of this format
+- Github
+  - projects are hosted at this publicly accessible collaboration platform which
+    is built around Git repositories
+
+This is a "just the steps" mini-tutorial (text with preceding `##` is either for
+you to replace, or instructions to follow). It is aimed at first-time users of
+Git, or relatively new users wanting to know the steps this project uses in the
+context of Github. If any step is confusing it should be easy to find details &
+explanations by searching Github documentation, or googling the command and
+especially looking at stackoverflow answers that come up for it. The
+instructions below are for the example-case of a first-time contributor adding a
+language to the translation-table:
+
+1. Ensure you have the [git][Git] commandline tool installed and usable in a
+   shell. If on Windows and you haven't already installed a "POSIX compatible"
+   shell ("command line"), the simplest thing is to install
+   [the bundled git package][Git windows bundle] which includes its own shell.
+1. In your browser, logged into Github, in the top-right corner of
+   [this repo][Project main page], click the `Fork` icon which will create your
+   own fork.
+1. In the shell do the following. Setting the variables at the beginning is just
+   to make this tutorial more readable, you can obviously enter the text
+   directly each time instead of using variables if you prefer:
+    ```sh
+    common_dir=## Some common directory to hold your repositories.
+    user=## Your github username.
+    lang=## Your translation language.
+
+    mkdir -p "${common_dir}"
+    cd "${common_dir}"
+    ```
+1. In the shell clone your fork to a local directory. For the simpler https
+   method (but which requires you to enter your password whenever you do remote
+   actions), do:
+    ```sh
+    git clone https://github.com/${user}/GoogleContactsEventsNotifier.git
+    ```
+    If you **optionally** want to be more advanced and use ssh instead of https
+    then do the documented steps for
+    [adding your ssh public-key to your github account][Add ssh key] if you
+    haven't already, and then do:
+    ```sh
+    git clone git@github.com:${user}/GoogleContactsEventsNotifier.git
+    ```
+1. In the shell do the following to create the new branch with your changes. You
+   will need to know which existing upstream branch it should be based on.
+   Although basing it on the upstream `master` branch is usually what you want,
+   sometimes you may need to base it on an "upstream feature/fix branch" which
+   the maintainers will later merge to their `master` branch (if in doubt ask
+   the maintainers):
+    ```sh
+    upstream_branch=## The upstream branch.
+
+    cd GoogleContactsEventsNotifier
+    git checkout -b ${lang}-translations ${upstream_branch}
+    ${EDITOR} code.gs

```

Comment:
I would replace this with a comment like `## Open code.gs in your text editor of choice and...` as I don't think most of the users who will need to follow this tutorial make use of a CLI text editor or know how to run their text editor from CLI.  
Having the comment instead of a command, however, should be fine for both those who use CLI editors and those who don't.

The same applies to lines 174 and 190.